langkit/tests/test_input_output.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import whylogs as why
+from whylogs.experimental.core.udf_schema import generate_udf_dataset_schema
+
+@pytest.fixture
+def interactions():
```

Comment:
Suggest we don’t use actual prompt/response pairs in unit test fixture, either use a dataset loaded from hugging face (integration test kind of thing), or use short strings? We aren’t testing the actual similarity or the quality of the models, let’s focus on testing the shape of the things passed to the udfs.